### PR TITLE
Create stubs for Universe v3 equivalents of Cosmos and MultiRepository

### PR DESCRIPTION
**Pull request by cruhland**

### CODE REVIEW

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala [hunk 1]
```diff
@@ -113,3 +113,16 @@ final class MultiRepository (
...
+// TODO (version): Delete this
+final class V3MultiRepository(
+  packageRepositoryStorage: PackageSourcesStorage,
+  universeClient: UniverseClient
+) extends V3PackageCollection {
+
+  override def getPackageByPackageVersion(
+    packageName: String,
+    packageVersion: Option[universe.v3.model.PackageDefinition.Version]
+  ): Future[(universe.v3.model.V3Package, Uri)] = ???
```
Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala]: Okay. After we merge the unverser v2 to universe v3 PR we should change this to `internal.model.PackageDefinition`.

**Comment by jsancio** [comment]
Is this PR ready for review? Correct me if I am wrong but this PR doesn't do what the title says, no?

**Comment by cruhland** [comment]
@jsancio I updated the title to be more accurate. The intention is to reduce the number of surprises when it comes time to integrate all our changes.

File: cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala (hunk 1 above)
Comment (jsancio) [inline cosmos-server/src/main/scala/com/mesosphere/cosmos/MultiRepository.scala]: This should return `Future[(internal.model.PackageDefinition, Uri)]`.

**Comment by jsancio** [comment]
LGTM after we address one small comment!

**Comment by cruhland** [comment]
@jsancio updated. Changing the type required some of the endpoint handlers to change, so you should take another look. Changes are in a separate commit.

**Comment by jsancio** [comment]
LGTM. I see that we are using encoders explicitly in a few places because of the `universe.v2` and `universe.v3` conflict. Type classes lose some of their appeal if we have to use them explicitly.

**Pull request merged**